tmva/tmva/CMakeLists.txt

```diff
@@ -222,6 +222,11 @@ else()
 endif()
 endif()
 
+if (vdt AND NOT builtin_vdt)
+  include_directories(${VDT_INCLUDE_DIRS})
+endif()
+
+
```

Comment:
If you are addressing the case Vdt is external, please try to use `target_include_directories()` to propagate the include to dependencies of TMVA.

Reply:
this would become
```
target_include_directories(TMVA PRIVATE ${VDT_INCLUDE_DIRS})
```
right?

Reply:
Yes, but you may need to make it PUBLIC if Vdt is used in TMVA headers.

Reply:
Also, it needs to come aftervthe target has been created in the file.